portal/server/routes/contract/bond/index.js

```diff
@@ -468,23 +467,15 @@ router.post('/contract/:_id/bond/:bondId/confirm-requested-cover-start-date', as
         }],
       };
     } else {
-      const previousCoverStartDate = moment().set({
+      const previousCoverStartDate = set(new Date(), {
         date: Number(bondToRender['requestedCoverStartDate-day']),
         month: Number(bondToRender['requestedCoverStartDate-month']) - 1, // months are zero indexed
         year: Number(bondToRender['requestedCoverStartDate-year']),
       });
 
-      const previousCoverStartDateTimestamp = moment(previousCoverStartDate).utc().valueOf().toString();
+      const previousCoverStartDateTimestamp = previousCoverStartDate.valueOf().toString();
 
-      const now = moment();
-
-      const dateOfCoverChange = moment().set({
-        date: Number(moment(now).format('DD')),
-        month: Number(moment(now).format('MM')) - 1, // months are zero indexed
-        year: Number(moment(now).format('YYYY')),
-      });
-
-      const dateOfCoverChangeTimestamp = moment(dateOfCoverChange).utc().valueOf().toString();
+      const today = new Date();
```

Comment:
This is where we can use a helper function which can return epoch or not based on my comments on the other related PR.

```javascript
const getNow = (epoch = false) => epoch ? new Date().valueOf() : new Date();
```

Reply:
FWIW I'd much prefer two functions, one of getNow and one of getNowAsEpoch, I think calling this function and having a boolean to swap between the two return types is probably a little confusing.

In terms of the bigger question of if this should be abstracted to a helper function, what's the use for this specific helper where it would just be 
`const getNow = () => new Date()`
I personally prefer not abstracting away what we're starting here and using variable naming to show what were doing with .valueOf or not, e.g. 
const today = new Date();
or
const todayAsEpoch = newDate().valueOf();